src/Validator/ValidatorInterface.php

```diff
@@ -21,7 +21,7 @@
      *
      * @return bool
      */
-    public function isValid();
+    public function __invoke();
```

Comment:
I'm not sure what else can be done. These things end up in `EventManager::triggerListeners()`, which does a `$listener($event)`, not a `$listener->isValid()` or a `call_user_func($listener)`. Suggestions?